src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.html

```diff
@@ -4,22 +4,24 @@
 >
   <div
     data-test-subj="field-{{::field.name}}"
+    tabindex="0"
+    role="button"
     ng-click="toggleDetails(field)"
+    ng-keyup="onKeyUp($event, field)"
     class="sidebar-item-title discover-sidebar-item"
   >
     <field-name
       class="discover-sidebar-item-label"
       field="field"
     ></field-name>
-    <div class="discover-sidebar-item-actions">
-      <button
-        ng-if="field.name !== '_source'"
-        ng-click="toggleDisplay(field)"
-        ng-class="::field.display ? 'kuiButton--danger' : 'kuiButton--primary'"
-        ng-bind="::field.display ? 'remove' : 'add'"
-        class="kuiButton kuiButton--small kuiButton--primary"
-        data-test-subj="fieldToggle-{{::field.name}}"
-      ></button>
-    </div>
+
+    <button
+      ng-if="field.name !== '_source'"
+      ng-click="toggleDisplay(field)"
+      ng-class="::field.display ? 'kuiButton--danger' : 'kuiButton--primary'"
+      ng-bind="::field.display ? 'remove' : 'add'"
+      class="discover-sidebar-item-action kuiButton kuiButton--small kuiButton--primary"
+      data-test-subj="fieldToggle-{{::field.name}}"
+    ></button>
```

Comment:
For some reason this button is not reacting to the space bar, even though it is a native button.